test/behaviors/t_for_all.clj

```diff
@@ -172,3 +175,87 @@
   (for-all
     [x gen/int]
     x => integer?))
+
+(gen-let [s (gen/return "s")]
```

Comment:
can you write examples of `gen-let` failing by creating `silent-gen-let` ([like this](https://github.com/marick/Midje/pull/416/files#diff-41075cd6b3c51c537586be18c8cdd597R48))
For example https://github.com/marick/Midje/pull/416/files#diff-d356953a2908279065444c22223da180R122